modules_core/com.smf.securewebservices/src/com/smf/securewebservices/SWSConfig.java

```diff
@@ -49,4 +53,7 @@ public String getPrivateKey() {
         return privateKey;
     }
 
+    public String getPublicKey() { return publicKey; }
```

Comment:
setters is not necessary?

Reply:
Not here... To set the fields, a js was made to generate and populate the records.